test/install_script/nvm_bash_completion

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+cleanup () {
+  unset -f die nvm_detect_profile
+  rm -f test_profile > /dev/null 2>&1
+}
+
+die () { echo $@ ; exit 1; }
+
+NVM_ENV=testing . ../../install.sh
+
+nvm_detect_profile () {
+  echo "$HOME/test_profile"
```

Comment:
in `cleanup`, you're removing `test_profile`, but here you return `$HOME/test_profile` - won't this leave behind a file in my home directory, slash modify it?